src/views/Header.vue

```diff
@@ -0,0 +1,105 @@
+
+
+<script>
+import {getAuthenticatedHeaders,logout} from "@/static_functions";
+export default {
+  emits: ['tendersOpen', 'offersOpen', 'newTenderOpen'],
+  data() {
+    return {
+      page: 0,
+      tenderCount: 0,
+      offerCount: 0,
+    }
+  },
+  async beforeCreate() {
+    this.tenderCount = await fetch('http://localhost:8080/tenders/me/count',{
+      method: 'GET',
+      node: 'cors',
+      cache: 'no-cache',
+      credentials: 'same-origin',
+      headers: getAuthenticatedHeaders(),
+    })
+        .then((response) => {
+              if (response.ok) {
+                return response.json()
+              } else if(response.status === 401 || response.status === 403){
+                logout()
+                return null;
+              }
+            })
+        .then(response=>
+    {
+      return response;
+    });
+    this.offerCount = await fetch('http://localhost:8080/tenders/me/offers/count',{
+      method: 'GET',
+          node: 'cors',
+          cache: 'no-cache',
+          credentials: 'same-origin',
+          headers: getAuthenticatedHeaders(),
+    })
+
+        .then((response) => {
+          if (response.ok) {
+            return response.json()
+          } else if(response.status === 401 || response.status === 403){
+            logout()
+            return null;
+          }
+        }
+        )
+        .then(response=>
+        {
+          return response;
+        });
+  },
+  methods: {
+
+    onLogout () {
+      logout();
+    } ,
+
+    onTendersClick() {
+      this.$emit("tendersOpen", true);
+    },
+    onOffersClick() {
+      this.$emit("offersOpen", true);
+    },
+    onCreateNewTender() {
+      this.$emit("newTenderOpen", true);
+    }
+  }
+}
+ </script>
+
+<template>
+  <header>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
better to remove this import because you already use different version of font-awesome.
I'd stay with Vuetify's icons